src/util/matcher/stitcher.js

```diff
@@ -0,0 +1,104 @@
+function stitcher(processedPoints, overlay, map) { // jshint ignore:line
+  var center, corners;
+  var sectors = {s00: [], s01: [], s10: [], s11: [], population: []};
+  for (var i in processedPoints.points) {
+    sectors.s00[i] = [];
+    sectors.s01[i] = [];
+    sectors.s10[i] = [];
+    sectors.s11[i] = [];
+    center = processedPoints.images[i].getCenter();
```

Comment:
Ah, so shouldn't `processedPoints` be a property of each image, rather than the other way around?

Reply:
Ah, i get it. `images[0]` and `images[1]` are the two origin images for the match? And is `processedPoints` actually better named `processedPairs`? Or `matchedPointPairs`?